conf/fileserver.conf

```diff
@@ -27,11 +27,44 @@
 
 # PERMISSIONS
 
-# Every static mount point should have an `allow *` line; setting more
-# granular permissions in this file is deprecated. Instead, you can
-# control file access in auth.conf by controlling the
-# /file_metadata(s)/<mount point> and /file_content(s)/<mount point> paths.
-# For example:
+# Setting permissions - for example, using the allow, allow_ip, or deny
+# directives - is no longer supported. Instead, you can control file access in
+# auth.conf by controlling the /file_metadata(s)/<mount point> and
+# /file_content(s)/<mount point> paths.
+#
+# For auth.conf functionality, it is recommended that you use Puppet Server as
+# your Puppet master, with the "jruby-puppet.use-legacy-auth-conf" setting
+# either set to "false" or, for Puppet Server 5 or later, not present.  Support
```

Comment:
Same note here about the use of "Support". Maybe it could be rephrased to cut off "Support for" and just be  "The legacy auth.conf format used by Ruby Puppet code is expected to eventually be dropped in favor of the HOCON-based auth.conf format that Puppet Server uses." ?